dbms/src/IO/createReadBufferFromFileBase.cpp

```diff
@@ -23,8 +25,12 @@ std::unique_ptr<ReadBufferFromFileBase> createReadBufferFromFileBase(const std::
 	}
 	else
 	{
+#ifndef __APPLE__
```

Comment:
I actually think that it'll be better to implement AIO on MacOS X in future, so that's why I don't want to ignore the settings.